apps/algorithm/search/README.md

```diff
@@ -2,10 +2,12 @@
 
 1. Clone repository.
 
-2. In martletplace directory run `docker-compose –build`. This step may take a while (few minutes to build) as it is as large application.
+2. In martletplace directory run `docker-compose down -v` to ensure no containers are running and any cached volumes are deleted.
```

Comment:
This doesn't work. `-v` can only find and delete the anonymous volumes if the stack is still running

Reply:
sadge, will update the readme to create instructions that will work.